src/ripple/nodestore/tests/import_test.cpp

```diff
@@ -748,7 +748,7 @@ class rekey_test : public beast::unit_test::suite
     }
 };
 
-BEAST_DEFINE_TESTSUITE_MANUAL(rekey,NodeStore,ripple);
+BEAST_DEFINE_TESTSUITE(rekey,NodeStore,ripple);
```

Comment:
The purpose of this test is to allow a command line to regenerate the .key file for a nudb nodestore. Marking it non-manual will just cause it to fail with "missing parameter" errors.